### PR TITLE
Updates to Illustrations section

### PR DESCRIPTION
**Pull request by charityshin**
- Fixes Links in Illustrations page
- Updates new Mascot

### CODE REVIEW

File: brand/identity/index.md [hunk 1]
```diff
@@ -190,57 +190,57 @@ layout: app
 <h3 class="m-bottom-2">Clanky</h3>
 	<div class="row">
 		<div class="col-sm-4 col-md-2">
-			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/mascot/m-1-1.png" alt="Dreamhost mascot" />
-			<p class="p-2 bg-c-g100">Format: <a href="{{site.baseurl}}/assets/downloads/mascot/m-1-1.eps">EPS</a></p>
+			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/mascot/new/m-1-1.png" alt="Dreamhost mascot" />
+			<p class="p-2 bg-c-g100">Format: <a href="{{site.baseurl}}/assets/images/mascot/new/m-1-1.png" download>PNG</a> | <a href="{{site.baseurl}}/assets/downloads/mascot/new/m-1-1.eps" download>EPS</a></p>
```
Comment (BillyH80) [inline brand/identity/index.md:194]: Is there a reason for having `download` as an attribute in the HTML tag here?

File: illustrations/assets/index.md [hunk 2]
```diff
@@ -11,35 +11,35 @@ layout: app
...
 			<p class="p-2 bg-c-g100 t-center"><strong>Domain</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-domain@2x.png" download>PNG</a></p>
 		</div>
 		<div class="col-12 col-md-3">
-			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-finance@2x.png" alt="Dreamhost Spot Illustration" />
+			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-finance@2x.png" alt="DH Finance Illustration" />
 			<p class="p-2 bg-c-g100 t-center"><strong>Finance</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-finance@2x.png" download>PNG</a></p>
 		</div>
 		<div class="col-12 col-md-3">
-			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-tech-support-1@2x.png" alt="Dreamhost Spot Illustration" />
+			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-tech-support-1@2x.png" alt="DH Tech Support 1" />
```
Comment (BillyH80) [inline illustrations/assets/index.md:30]: Great attention to detail, and thanks for giving these unique alt tags!

File: brand/identity/index.md (hunk 1 above)
Comment (charityshin) [reply brand/identity/index.md:194]: When I remove the attribute, it does show the image in a new tab. Should we have `download` attribute only for EPS?

Comment (charityshin) [reply brand/identity/index.md:194]: I think the images were always "right-click" > "Save Image" but not many people knew. I assume people like having an obvious "download link"

Comment (charityshin) [reply brand/identity/index.md:194]: Oh wait sorry this is for Identity page

**Review by BillyH80 — APPROVED** [review]

**Pull request merged**